### PR TITLE
Renaming pyFAST to openfast_toolbox  (see #38 and #60)

### PR DESCRIPTION
**Pull request by ebranlard**
(see #38 and #60)
 
This pull request: 
- renames the package to `openfast_toolbox`, to avoid conflict with the repository  `pyFAST` on `pypi/pip`
- the `input_output` folder is renamed to `.io` 
- the pull request adds developper notes at the root of the folder for new release / pypi updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,21 +22,21 @@ pytest
...
 ## QuickStart and main usage
 
 ### Read and write files
-Find examples scripts in this [folder](pyFAST/input_output/examples) and the different fileformats [here](pyFAST/input_output). 
+Find examples scripts in this [folder](openfast_toolbox/input_output/examples) and the different fileformats [here](openfast_toolbox/input_output). 
 
 Read an AeroDyn file (or any OpenFAST input file), modifies some values and write the modified file:
 ```python
-from pyFAST.input_output import FASTInputFile
+from openfast_toolbox.input_output import FASTInputFile
```
Comment (rthedin) [inline README.md]: Shouldn't this be `openfast_toolbox.io`?

File: README.md [hunk 2]
```diff
@@ -46,15 +46,15 @@ f.write('AeroDyn_Changed.dat')
 
 Read an OpenFAST binary output file and convert it to a pandas DataFrame
 ```python
-from pyFAST.input_output import FASTOutputFile
+from openfast_toolbox.input_output import FASTOutputFile
```
Comment (rthedin) [inline README.md]: Ditto

File: README.md [hunk 3]
```diff
@@ -46,15 +46,15 @@ f.write('AeroDyn_Changed.dat')
...
 df = FASTOutputFile('5MW.outb').toDataFrame()
 time  = df['Time_[s]']
 Omega = df['RotSpeed_[rpm]']
 ```
 
 Read a TurbSim binary file, modify it and write it back
 ```python 
-from pyFAST.input_output import TurbSimFile
+from openfast_toolbox.input_output import TurbSimFile
```
Comment (rthedin) [inline README.md]: Ditto

File: openfast_toolbox/io/README.md [hunk 4]
```diff
@@ -1,63 +1,63 @@
...
+
+
+## Examples
+Examples scripts are found in this [folder](examples). 
+Below are simple examples to get started:
+
+
+Read an AeroDyn file, modifies some values and write the modified file:
+```python
+from openfast_toolbox.input_output import FASTInputFile
```
Comment (rthedin) [inline openfast_toolbox/io/README.md:36]: Fix `input_output` here too.

File: openfast_toolbox/io/README.md [hunk 5]
```diff
@@ -1,63 +1,63 @@
...
+filename = 'AeroDyn.dat'
+f = FASTInputFile(filename)
+f['TwrAero'] = True
+f['AirDens'] = 1.225
+f.write('AeroDyn_Changed.dat')
+```
+
+Read an OpenFAST binary output file and convert it to a pandas DataFrame
+```python
+from openfast_toolbox.input_output import FASTOutputFile
```
Comment (rthedin) [inline openfast_toolbox/io/README.md:46]: and here

File: openfast_toolbox/io/README.md [hunk 6]
```diff
@@ -1,63 +1,63 @@
...
+df = FASTOutputFile('5MW.outb').toDataFrame()
+time  = df['Time_[s]']
+Omega = df['RotSpeed_[rpm]']
+```
+
+Read a TurbSim binary file
+```python 
+from openfast_toolbox.input_output import TurbSimFile
```
Comment (rthedin) [inline openfast_toolbox/io/README.md:54]: and here

**Review by rthedin — COMMENTED** [review]
Looks good to me. You left behind a few `input_output`. Do a last `grep` on that before merging in.

**Comment by ebranlard** [comment]
Thank you for the review, I'm sorry github had a bug and somehoe didn't update the files in time for your review.

**Pull request merged**